index.html

```diff
@@ -136,6 +136,7 @@ <h1>TourGuide . . .</h1>
   </nav> -->
     <div class="nav-container" id="top">
       <nav class="newNav">
+        <img src="C:\Users\user\Desktop\tlogo\Travel_Website\img\A T logo with a white background and a square shape, connected to nature for a travel company.png" height="30" style="border-radius:50%;width:30px;">

```

Comment:
![image](https://github.com/apu52/Travel_Website/assets/114172928/3a9e26ac-42ac-4807-a214-04939bbbcdf9)

the logo can't be seen...maybe there is a img name problem , try to look at it